packages/g/glew/xmake.lua

```diff
@@ -11,8 +11,29 @@ package("glew")
         add_versions("2.1.0", "2700383d4de2455f06114fbaf872684f15529d4bdc5cdea69b5fb0e9aa7763f1")
     end
 
-    on_load("windows", "mingw", function (package)
-        package:add("links", is_plat("windows") and "glew32s" or "glew32")
+    if is_plat("windows", "mingw") then
+        add_syslinks("glu32", "opengl32")
+    else
+        add_links("GLEW")
+        if is_plat("linux") then
+            add_syslinks("GLU", "GL")
+        elseif is_plat("macosx") then
+            add_frameworks("OpenGL")
+        end
+    end
+
+    on_load(function (package)
+        if package:config("shared") then
+            package:add("defines", "GLEW_BUILD")
+            if package:is_plat("windows", "mingw") then
+                package:add("links", "glew32")
+            end
+        else
+            package:add("defines", "GLEW_STATIC")
+            if package:is_plat("windows", "mingw") then
+                package:add("links", "glew32s")
```

Comment:
and here. syslinks

Reply:
The syslinks are always the same. So I added them above using ```add_syslinks```

Reply:
You can use add_syslinks to instead of `package:add("links")` or use `package:add("syslinks")`

Reply:
But glew32 is not a syslink.

Reply:
Oh, glew32 is not. my mistake.